hack/build-images.sh

```diff
@@ -30,9 +30,14 @@ if [[ "${OS_RELEASE:-}" == "n" ]]; then
   OS_BUILD_PLATFORMS=("${OS_IMAGE_COMPILE_PLATFORMS[@]-}")
 
   echo "Building images from source ${OS_RELEASE_COMMIT}:"
-	echo
+  echo
   os::build::build_static_binaries "${OS_IMAGE_COMPILE_TARGETS[@]-}" "${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]-}"
-	os::build::place_bins "${OS_IMAGE_COMPILE_BINARIES[@]}"
+  # Build dockerregistry
+  (
+    export GO15VENDOREXPERIMENT=1
+    os::build::build_static_binaries "${OS_IMAGE_COMPILE_DOCKERREGISTRY_TARGETS[@]-}"
+  )
+  os::build::place_bins "${OS_IMAGE_COMPILE_BINARIES[@]}"
```

Comment:
No, they're not (https://github.com/openshift/origin/pull/8938/commits/084a33d8949b2810e9b1031593c9eb1df8d77407#diff-f4e81489ff94a80b6c06c769fac10d80R50). And IMHO they should.